app/models/blacklisted_event.rb

```diff
@@ -1,10 +1,11 @@
 class BlacklistedEvent < ApplicationRecord
   belongs_to        :ext_management_system, :foreign_key => "ems_id"
 
-  default_value_for :enabled, true
-  after_save        :reload_all_server_settings
-  after_destroy     :reload_all_server_settings, :audit_deletion
+  attribute :enabled, :default => true
+  after_validation  :log_enabling, :if => :enabled_changed?
   after_create      :audit_creation
+  after_destroy     :reload_all_server_settings, :audit_deletion
+  after_save        :reload_all_server_settings
```

Comment:
the cop wanted these sorted by their order... it's only the after_validation that's new